spec/services/rate_limit_checker_spec.rb

```diff
@@ -5,11 +5,34 @@
   let(:article) { create(:article, user: user) }
   let(:rate_limit_checker) { described_class.new(user) }
 
+  def cache_key(action)
+    rate_limit_checker.send("limit_cache_key", action)
+  end
+
   describe "#limit_by_action" do
     it "returns false for invalid action" do
       expect(rate_limit_checker.limit_by_action("random-nothing")).to be(false)
     end
 
+    # published_article_creation limit we check against the databse rather than our cache
```

Comment:
small typo

Reply:
That took me reading this twice to figure it out 😂